sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -938,12 +938,12 @@ def update_issuer(self, issuer_name, **kwargs):
         if admin_contacts and admin_contacts[0]:
```

Comment:
Comment about consequences of `admin_contacts` being a rando iterable applies here as well.

Reply:
rip, doing that now